modules/openapi-generator/src/main/resources/typescript-fetch/runtimeSagasAndRecords.mustache

```diff
@@ -0,0 +1,115 @@
+/* tslint:disable */
+/* eslint-disable */
+
+import {fromJS as originalFromJS, isIndexed, List, Map as ImmMap, RecordOf} from 'immutable';
+import {normalize, NormalizedSchema, schema, Schema} from "normalizr";
+import {ActionDefinition, createAction} from "redux-ts-simple";
+
+export const knownRecordFactories = new Map<string, any>();
+export const knownIndexedSetByKey = [];
+
+export function appFromJS(any: any): any {
+	return originalFromJS(any, (key, value) => {
```

Comment:
please review all your file and make sure you use a "4 space tabs" instead of "real tabs". this is the standard in this project

Reply:
oh...